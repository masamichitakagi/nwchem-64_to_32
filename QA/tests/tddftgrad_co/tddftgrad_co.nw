# Test case for the TDHF/RPA analytical excited state gradient 
# functionality using the CO molecule for triplet states.

echo

start tddftgrad_co

geometry
C     0.00000000     0.00000000    -0.64628342
O     0.00000000     0.00000000     0.48264375
symmetry c1
end

basis spherical
 * library "3-21G"
end

dft
 xc HFExch 1.0
 direct
end

tddft
 nroots 3
 nosinglet
 triplet
 target 1
 civecs 
 grad 
  root 1
 end
end

task tddft gradient
