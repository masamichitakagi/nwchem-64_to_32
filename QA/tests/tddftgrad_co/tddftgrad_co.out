 argument  1 = tddftgrad_co.nw



============================== echo of input deck ==============================
# Test case for the TDHF/RPA analytical excited state gradient
# functionality using the CO molecule for triplet states.

echo

start tddftgrad_co

geometry
C     0.00000000     0.00000000    -0.64628342
O     0.00000000     0.00000000     0.48264375
symmetry c1
end

basis spherical
 * library "3-21G"
end

dft
 xc HFExch 1.0
 direct
end

tddft
 nroots 3
 nosinglet
 triplet
 target 1
 civecs
 grad
  root 1
 end
end

task tddft gradient
================================================================================


                                         
                                         


              Northwest Computational Chemistry Package (NWChem) 6.5
              ------------------------------------------------------


                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352

                              Copyright (c) 1994-2014
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute

             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file

                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname        = orion
    program         = ../../../bin/LINUX64/nwchem
    date            = Tue Sep  2 17:21:15 2014

    compiled        = Sat_Aug_30_06:49:11_2014
    source          = /home/niri/nwchem/nwchem-6.5
    nwchem branch   = 6.5
    nwchem revision = 26025
    ga revision     = 10501
    input           = tddftgrad_co.nw
    prefix          = tddftgrad_co.
    data base       = ./tddftgrad_co.db
    status          = startup
    nproc           =        4
    time left       =     -1s



           Memory information
           ------------------

    heap     =   13107196 doubles =    100.0 Mbytes
    stack    =   13107201 doubles =    100.0 Mbytes
    global   =   26214400 doubles =    200.0 Mbytes (distinct from heap & stack)
    total    =   52428797 doubles =    400.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------

  0 permanent = .
  0 scratch   = .




                                NWChem Input Module
                                -------------------



 Scaling coordinates for geometry "geometry" by  1.889725989
 (inverse scale =  0.529177249)

 Turning off AUTOSYM since
 SYMMETRY directive was detected!


          ------
          auto-z
          ------


                             Geometry "geometry" -> ""
                             -------------------------

 Output coordinates in angstroms (scale by  1.889725989 to convert to a.u.)

  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 C                    6.0000     0.00000000     0.00000000    -0.64510124
    2 O                    8.0000     0.00000000     0.00000000     0.48382593

      Atomic Mass 
      ----------- 

      C                 12.000000
      O                 15.994910


 Effective nuclear repulsion energy (a.u.)      22.4996869833

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
     0.0000000000     0.0000000000     0.0000000000



                                Z-matrix (autoz)
                                -------- 

 Units are Angstrom for bonds and degrees for angles

      Type          Name      I     J     K     L     M      Value
      ----------- --------  ----- ----- ----- ----- ----- ----------
    1 Stretch                  1     2                       1.12893


            XYZ format geometry
            -------------------
     2
 geometry
 C                     0.00000000     0.00000000    -0.64510124
 O                     0.00000000     0.00000000     0.48382593

 ==============================================================================
                                internuclear distances
 ------------------------------------------------------------------------------
       center one      |      center two      | atomic units |  angstroms
 ------------------------------------------------------------------------------
    2 O                |   1 C                |     2.13336  |     1.12893
 ------------------------------------------------------------------------------
                         number of included internuclear distances:          1
 ==============================================================================



  library name resolved from: environment
  library file name is: </home/niri/nwchem/nwchem-6.5/src/basis/libraries/>
  


 Summary of "ao basis" -> "" (spherical)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 *                           3-21G                    on all atoms 


 tddft_input: filename not found; default name will be used

                                 NWChem DFT Module
                                 -----------------


                      Basis "ao basis" -> "ao basis" (spherical)
                      -----
  C (Carbon)
  ----------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  1.72256000E+02  0.061767
  1 S  2.59109000E+01  0.358794
  1 S  5.53335000E+00  0.700713

  2 S  3.66498000E+00 -0.395897
  2 S  7.70545000E-01  1.215840

  3 P  3.66498000E+00  0.236460
  3 P  7.70545000E-01  0.860619

  4 S  1.95857000E-01  1.000000

  5 P  1.95857000E-01  1.000000

  O (Oxygen)
  ----------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  3.22037000E+02  0.059239
  1 S  4.84308000E+01  0.351500
  1 S  1.04206000E+01  0.707658

  2 S  7.40294000E+00 -0.404453
  2 S  1.57620000E+00  1.221560

  3 P  7.40294000E+00  0.244586
  3 P  1.57620000E+00  0.853955

  4 S  3.73684000E-01  1.000000

  5 P  3.73684000E-01  1.000000



 Summary of "ao basis" -> "ao basis" (spherical)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 C                           3-21G                   5        9   3s2p
 O                           3-21G                   5        9   3s2p




 Summary of "ao basis" -> "ao basis" (spherical)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 C                           3-21G                   5        9   3s2p
 O                           3-21G                   5        9   3s2p


  Caching 1-el integrals 

            General Information
            -------------------
          SCF calculation type: DFT
          Wavefunction type:  closed shell.
          No. of atoms     :     2
          No. of electrons :    14
           Alpha electrons :     7
            Beta electrons :     7
          Charge           :     0
          Spin multiplicity:     1
          Use of symmetry is: off; symmetry adaption is: off
          Maximum number of iterations:  30
          This is a Direct SCF calculation.
          AO basis - number of functions:    18
                     number of shells:    10
          Convergence on energy requested: 1.00D-06
          Convergence on density requested: 1.00D-05
          Convergence on gradient requested: 5.00D-04

              XC Information
              --------------
                     Hartree-Fock (Exact) Exchange  1.000          

          Convergence Information
          -----------------------
          Convergence aids based upon iterative change in 
          total energy or number of iterations. 
          Levelshifting, if invoked, occurs when the 
          HOMO/LUMO gap drops below (HL_TOL): 1.00D-02
          DIIS, if invoked, will attempt to extrapolate 
          using up to (NFOCK): 10 stored Fock matrices.

                    Damping( 0%)  Levelshifting(0.5)       DIIS
                  --------------- ------------------- ---------------
          dE  on:    start            ASAP                start   
          dE off:    2 iters         30 iters            30 iters 


      Screening Tolerance Information
      -------------------------------
          Density screening/tol_rho: 1.00D-10
          AO Gaussian exp screening on grid/accAOfunc:  14
          CD Gaussian exp screening on grid/accCDfunc:  20
          XC Gaussian exp screening on grid/accXCfunc:  20
          Schwarz screening/accCoul: 1.00D-08


      Superposition of Atomic Density Guess
      -------------------------------------

 Sum of atomic energies:        -111.83245401

      Non-variational initial energy
      ------------------------------

 Total energy =    -112.280932
 1-e energy   =    -197.166614
 2-e energy   =      62.385995
 HOMO         =      -0.371277
 LUMO         =       0.065454

   Time after variat. SCF:      0.1
   Time prior to 1st pass:      0.1

           Memory utilization after 1st SCF pass: 
           Heap Space remaining (MW):       13.11            13106012
          Stack Space remaining (MW):       13.11            13106980

   convergence    iter        energy       DeltaE   RMS-Dens  Diis-err    time
 ---------------- ----- ----------------- --------- --------- ---------  ------
 d= 0,ls=0.0,diis     1   -112.0614069145 -1.35D+02  2.31D-02  2.72D-01     0.1
 d= 0,ls=0.0,diis     2   -112.0751282401 -1.37D-02  1.02D-02  2.12D-01     0.1
 d= 0,ls=0.0,diis     3   -112.0931393599 -1.80D-02  1.60D-03  1.08D-03     0.1
 d= 0,ls=0.0,diis     4   -112.0932923039 -1.53D-04  3.57D-04  3.87D-05     0.1
 d= 0,ls=0.0,diis     5   -112.0932986756 -6.37D-06  8.06D-05  1.86D-06     0.1
 d= 0,ls=0.0,diis     6   -112.0932989659 -2.90D-07  2.48D-05  8.63D-08     0.1
 d= 0,ls=0.0,diis     7   -112.0932989837 -1.78D-08  3.35D-06  1.80D-09     0.2


         Total DFT energy =     -112.093298983723
      One electron energy =     -197.100986698328
           Coulomb energy =       75.779719266911
    Exchange-Corr. energy =      -13.271718535644
 Nuclear repulsion energy =       22.499686983338

 Numeric. integr. density =        0.000000000000

     Total iterative time =      0.1s



                       DFT Final Molecular Orbital Analysis
                       ------------------------------------

 Vector    1  Occ=2.000000D+00  E=-2.057896D+01
              MO Center= -3.2D-17,  2.1D-17,  4.8D-01, r^2= 1.5D-02
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    10      0.983611  2 O  s          

 Vector    2  Occ=2.000000D+00  E=-1.131771D+01
              MO Center=  1.1D-17,  1.3D-17, -6.4D-01, r^2= 2.8D-02
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1      0.986176  1 C  s          

 Vector    3  Occ=2.000000D+00  E=-1.547669D+00
              MO Center= -1.1D-16, -5.9D-17,  1.6D-01, r^2= 4.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    15      0.666671  2 O  s                 10     -0.216959  2 O  s          
    11      0.212382  2 O  s                  5      0.185326  1 C  pz         
    14     -0.157135  2 O  pz         

 Vector    4  Occ=2.000000D+00  E=-7.853524D-01
              MO Center=  2.8D-17, -2.8D-16,  5.0D-01, r^2= 7.5D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    15      0.504804  2 O  s                 14      0.403918  2 O  pz         
    18      0.363858  2 O  pz                 6     -0.252801  1 C  s          
     2     -0.191383  1 C  s          

 Vector    5  Occ=2.000000D+00  E=-6.347788D-01
              MO Center=  3.4D-16,  2.2D-15,  2.2D-01, r^2= 8.1D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    17      0.480211  2 O  py                13      0.440488  2 O  py         
     4      0.249078  1 C  py                 8      0.197061  1 C  py         

 Vector    6  Occ=2.000000D+00  E=-6.347788D-01
              MO Center=  7.4D-16, -1.2D-16,  2.2D-01, r^2= 8.1D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    16      0.480211  2 O  px                12      0.440488  2 O  px         
     3      0.249078  1 C  px                 7      0.197061  1 C  px         

 Vector    7  Occ=2.000000D+00  E=-5.432414D-01
              MO Center= -1.0D-15, -1.6D-15, -8.8D-01, r^2= 9.5D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     6      0.775806  1 C  s                  5     -0.384098  1 C  pz         
    18      0.242003  2 O  pz                14      0.221158  2 O  pz         
     9     -0.173606  1 C  pz                 1     -0.150856  1 C  s          

 Vector    8  Occ=0.000000D+00  E= 1.617214D-01
              MO Center= -2.7D-17,  9.9D-17, -4.5D-01, r^2= 1.4D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     8      0.723555  1 C  py                17     -0.501704  2 O  py         
     4      0.384202  1 C  py                13     -0.291268  2 O  py         
     7     -0.199219  1 C  px         

 Vector    9  Occ=0.000000D+00  E= 1.617214D-01
              MO Center=  1.0D-16,  2.8D-17, -4.5D-01, r^2= 1.4D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     7      0.723555  1 C  px                16     -0.501704  2 O  px         
     3      0.384202  1 C  px                12     -0.291268  2 O  px         
     8      0.199219  1 C  py         

 Vector   10  Occ=0.000000D+00  E= 4.922346D-01
              MO Center= -1.2D-16,  2.8D-16, -6.0D-01, r^2= 2.2D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    15     -1.724538  2 O  s                  9      1.677565  1 C  pz         
     6      1.451153  1 C  s                 18      0.656675  2 O  pz         

 Vector   11  Occ=0.000000D+00  E= 9.263262D-01
              MO Center=  5.6D-16, -9.8D-17, -6.9D-01, r^2= 1.4D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     3      1.063562  1 C  px                 7     -0.996724  1 C  px         
     4     -0.187923  1 C  py                 8      0.176113  1 C  py         

 Vector   12  Occ=0.000000D+00  E= 9.263262D-01
              MO Center=  7.0D-17,  4.0D-16, -6.9D-01, r^2= 1.4D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     4      1.063562  1 C  py                 8     -0.996724  1 C  py         
     3      0.187923  1 C  px                 7     -0.176113  1 C  px         

 Vector   13  Occ=0.000000D+00  E= 1.094807D+00
              MO Center= -7.6D-16, -1.1D-15, -8.8D-01, r^2= 1.2D+00
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     5      1.124736  1 C  pz                 6      1.021553  1 C  s          
    18      0.608882  2 O  pz                 9     -0.479613  1 C  pz         
     2     -0.472634  1 C  s                 15     -0.418651  2 O  s          

 Vector   14  Occ=0.000000D+00  E= 1.348355D+00
              MO Center=  1.7D-17,  3.7D-16, -2.9D-01, r^2= 9.2D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2      1.471998  1 C  s                  6     -0.979070  1 C  s          
     5      0.506685  1 C  pz                14      0.343598  2 O  pz         
    18      0.335345  2 O  pz                 9     -0.183162  1 C  pz         

 Vector   15  Occ=0.000000D+00  E= 1.784091D+00
              MO Center= -9.9D-15,  5.3D-15,  8.2D-01, r^2= 6.8D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    18      1.333847  2 O  pz                14     -0.990712  2 O  pz         
     2      0.445800  1 C  s                 15     -0.288955  2 O  s          
     6      0.278148  1 C  s                  9      0.209654  1 C  pz         
    11     -0.155365  2 O  s          

 Vector   16  Occ=0.000000D+00  E= 1.798448D+00
              MO Center=  1.0D-14, -5.6D-15,  4.8D-01, r^2= 8.5D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    16     -0.975604  2 O  px                12      0.903685  2 O  px         
    17      0.548126  2 O  py                13     -0.507720  2 O  py         
     7      0.307166  1 C  px                 8     -0.172576  1 C  py         

 Vector   17  Occ=0.000000D+00  E= 1.798448D+00
              MO Center=  4.0D-16,  7.1D-16,  4.8D-01, r^2= 8.5D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
    17     -0.975604  2 O  py                13      0.903685  2 O  py         
    16     -0.548126  2 O  px                12      0.507720  2 O  px         
     8      0.307166  1 C  py                 7      0.172576  1 C  px         


 center of mass
 --------------
 x =   0.00000000 y =   0.00000000 z =  -0.00016624

 moments of inertia (a.u.)
 ------------------
          31.204231660766           0.000000000000           0.000000000000
           0.000000000000          31.204231660766           0.000000000000
           0.000000000000           0.000000000000           0.000000000000

     Multipole analysis of the density
     ---------------------------------

     L   x y z        total         alpha         beta         nuclear
     -   - - -        -----         -----         ----         -------
     0   0 0 0      0.000000     -7.000000     -7.000000     14.000000

     1   1 0 0      0.000000      0.000000      0.000000      0.000000
     1   0 1 0      0.000000      0.000000      0.000000      0.000000
     1   0 0 1     -0.156250     -0.078125     -0.078125      0.000000

     2   2 0 0     -7.353699     -3.676849     -3.676849      0.000000
     2   1 1 0      0.000000      0.000000      0.000000      0.000000
     2   1 0 1      0.000000      0.000000      0.000000      0.000000
     2   0 2 0     -7.353699     -3.676849     -3.676849      0.000000
     2   0 1 1      0.000000      0.000000      0.000000      0.000000
     2   0 0 2     -9.506147    -12.555195    -12.555195     15.604244

                                NWChem TDDFT Module
                                -------------------


            General Information
            -------------------
           No. of orbitals :    36
            Alpha orbitals :    18
             Beta orbitals :    18
        Alpha frozen cores :     0
         Beta frozen cores :     0
     Alpha frozen virtuals :     0
      Beta frozen virtuals :     0
         Spin multiplicity :     1
    Number of AO functions :    18
        Use of symmetry is : off
      Symmetry adaption is : off
         Schwarz screening : 0.10D-07

              XC Information
              --------------
              Hartree-Fock (Exact) Exchange   1.00          

             TDDFT Information
             -----------------
          Calculation type : TDHF              
         Wavefunction type : Restricted triplets
          No. of electrons :    14
           Alpha electrons :     7
            Beta electrons :     7
              No. of roots :     3
          Max subspacesize :  4600
            Max iterations :   100
               Target root :     1
           Target symmetry : none
      Symmetry restriction : off
                 Algorithm : Optimal
        Davidson threshold : 0.10D-03

            Memory Information
            ------------------
          Available GA space size is         104857276 doubles
          Available MA space size is          26213252 doubles
          Length of a trial vector is           77
          Algorithm : Incore multiple tensor contraction
          Estimated peak GA usage is           1426334 doubles
          Estimated peak MA usage is              6600 doubles

    3 smallest eigenvalue differences (eV) 
--------------------------------------------------------
  No. Spin  Occ  Vir  Irrep   E(Occ)    E(Vir)   E(Diff)
--------------------------------------------------------
    1    1    7    8 a        -0.543     0.162    19.183
    2    1    7    9 a        -0.543     0.162    19.183
    3    1    6    8 a        -0.635     0.162    21.674
--------------------------------------------------------

  Entering Davidson iterations
  Restricted triplet excited states

  Iter   NTrls   NConv    DeltaV     DeltaE      Time   
  ----  ------  ------  ---------  ---------  --------- 
    1      3       0     0.21E+00   0.10+100        0.0
    2      9       0     0.75E-01   0.60E-01        0.1
    3     15       0     0.14E+00   0.12E-01        0.1
    4     21       0     0.32E-01   0.43E-01        0.1
    5     27       0     0.79E-02   0.56E-03        0.1
    6     33       2     0.18E-02   0.18E-04        0.1
    7     35       2     0.16E-03   0.48E-06        0.0
    8     37       3     0.36E-04   0.58E-08        0.1
  ----  ------  ------  ---------  ---------  --------- 
  Convergence criterion met

  Ground state a        -112.093298983723 a.u.

  ----------------------------------------------------------------------------
  Root   1 triplet a              0.202688187 a.u.                5.5154 eV 
  ----------------------------------------------------------------------------
     Transition Moments                    Spin forbidden
     Oscillator Strength                   Spin forbidden

     Occ.    7  a   ---  Virt.    8  a    0.12686 X
     Occ.    7  a   ---  Virt.    9  a    0.99423 X
     Occ.    7  a   ---  Virt.    9  a    0.15038 Y
     Occ.    7  a   ---  Virt.   11  a    0.12202 X
     Occ.    7  a   ---  Virt.   12  a    0.07830 X
  ----------------------------------------------------------------------------
  Root   2 triplet a              0.202688187 a.u.                5.5154 eV 
  ----------------------------------------------------------------------------
     Transition Moments                    Spin forbidden
     Oscillator Strength                   Spin forbidden

     Occ.    7  a   ---  Virt.    8  a    0.99423 X
     Occ.    7  a   ---  Virt.    8  a    0.15038 Y
     Occ.    7  a   ---  Virt.    9  a   -0.12686 X
     Occ.    7  a   ---  Virt.   11  a   -0.07830 X
     Occ.    7  a   ---  Virt.   12  a    0.12202 X
  ----------------------------------------------------------------------------
  Root   3 triplet a              0.227277500 a.u.                6.1845 eV 
  ----------------------------------------------------------------------------
     Transition Moments                    Spin forbidden
     Oscillator Strength                   Spin forbidden

     Occ.    3  a   ---  Virt.   10  a   -0.05150 X
     Occ.    5  a   ---  Virt.    8  a   -0.67185 X
     Occ.    5  a   ---  Virt.    8  a   -0.20946 Y
     Occ.    5  a   ---  Virt.    9  a   -0.30395 X
     Occ.    5  a   ---  Virt.    9  a   -0.09476 Y
     Occ.    5  a   ---  Virt.   12  a   -0.07179 X
     Occ.    5  a   ---  Virt.   17  a    0.06889 X
     Occ.    6  a   ---  Virt.    8  a    0.30395 X
     Occ.    6  a   ---  Virt.    8  a    0.09476 Y
     Occ.    6  a   ---  Virt.    9  a   -0.67185 X
     Occ.    6  a   ---  Virt.    9  a   -0.20946 Y
     Occ.    6  a   ---  Virt.   11  a   -0.07179 X
     Occ.    6  a   ---  Virt.   16  a    0.06889 X

              Target root =      1
          Target symmetry = none
      Ground state energy =   -112.093298983723
        Excitation energy =      0.202688187049
     Excited state energy =   -111.890610796674

 fn_civecs: ./tddftgrad_co.civecs_triplet                                                                                                                                                                                                                                  

  CI vectors are stored in ./tddftgrad_co.civecs_triplet   


                           NWChem TDDFT Gradient Module
                           ----------------------------


 ./tddftgrad_co.civecs_triplet                                                                                                                                                                                                                                  
 Calculated gradients of:
 Number of Triplet roots   1
 Triplet roots    1

 Start at time  cpu:        0.7s  wall:        1.1s

TDDFT Energy Check(  1) =       0.20268818704888


Iterative solution of linear equations
  No. of variables       77
  No. of equations        1
  Maximum subspace       40
        Iterations      250
       Convergence  1.0D-04
        Start time      1.3


   iter   nsub   residual    time
   ----  ------  --------  ---------
     1      1    1.48D-01       1.4
     2      2    6.06D-02       1.4
     3      3    1.37D-02       1.4
     4      4    2.58D-03       1.5
     5      5    4.04D-04       1.5
     6      6    7.84D-05       1.6
     Dipole Moment    X  0.00000   Y  0.00000   Z -0.95124

     No. of electrons (tr(P*S)):     0.1400000E+02



                         Root   1

                         TDDFT ENERGY GRADIENTS

    atom               coordinates                        gradient
                 x          y          z           x          y          z
   1 C       0.000000   0.000000  -1.219065    0.000000   0.000000   0.202729
   2 O       0.000000   0.000000   0.914298    0.000000   0.000000  -0.202729

 TDDFT Gradient time  cpu:        0.6s  wall:        0.7s

 Task  times  cpu:        1.2s     wall:        1.6s


                                NWChem Input Module
                                -------------------


 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls:  954      954     1.31e+05 8711     5.26e+04  128        0     1186     
number of processes/call 1.02e+00 1.25e+00 1.01e+00 2.00e+00 0.00e+00
bytes total:             1.03e+07 1.49e+06 4.28e+06 7.35e+04 0.00e+00 9.49e+03
bytes remote:            3.62e+05 2.46e+05 1.55e+05 -7.34e+04 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 2269880 bytes

MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        20	        56
	current total bytes		         0	         0
	maximum total bytes		     80136	  22510376
	maximum total K-bytes		        81	     22511
	maximum total M-bytes		         1	        23


                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:

                 M. Valiev, E.J. Bylaska, N. Govind, K. Kowalski,
              T.P. Straatsma, H.J.J. van Dam, D. Wang, J. Nieplocha,
                        E. Apra, T.L. Windus, W.A. de Jong
                 "NWChem: a comprehensive and scalable open-source
                  solution for large scale molecular simulations"
                      Comput. Phys. Commun. 181, 1477 (2010)
                           doi:10.1016/j.cpc.2010.04.018

                              AUTHORS & CONTRIBUTORS
                              ----------------------
          E. Apra, E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski,
       T. P. Straatsma, M. Valiev, H. J. J. van Dam, D. Wang, T. L. Windus,
    J. Hammond, J. Autschbach, K. Bhaskaran-Nair, J. Brabec, K. Lopata, W. Ma,
       M. Klemm, O. Villa, F. Aquino, S. Hirata, M. T. Hackler, T. Risthaus,
       M. Malagoli, A. Marenich, A. Otero-de-la-Roza, J. Mullin, P. Nichols,
      R. Peverati, J. Pittner, Y. Zhao, P.-D. Fan, A. Fonari, R. J. Harrison,
      M. Dupuis, D. Silverstein, D. M. A. Smith, J. Nieplocha, V. Tipparaju,
    M. Krishnan, B. E. Van Kuiken, A. Vazquez-Mayagoitia, L. Jensen, M. Swart,
      Q. Wu, T. Van Voorhis, A. A. Auer, M. Nooijen, L. D. Crosby, E. Brown,
      G. Cisneros, G. I. Fann, H. Fruchtl, J. Garza, K. Hirao, R. A. Kendall,
      J. A. Nichols, K. Tsemekhman, K. Wolinski, J. Anchell, D. E. Bernholdt,
      P. Borowski, T. Clark, D. Clerc, H. Dachsel, M. J. O. Deegan, K. Dyall,
    D. Elwood, E. Glendening, M. Gutowski, A. C. Hess, J. Jaffe, B. G. Johnson,
     J. Ju, R. Kobayashi, R. Kutteh, Z. Lin, R. Littlefield, X. Long, B. Meng,
      T. Nakajima, S. Niu, L. Pollack, M. Rosing, K. Glaesemann, G. Sandrone,
      M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe, A. T. Wong, Z. Zhang.

 Total times  cpu:        1.2s     wall:        1.9s
